src/Servers/HttpSys/src/HttpSysListener.cs

```diff
@@ -77,9 +77,9 @@ public HttpSysListener(HttpSysOptions options, ILoggerFactory loggerFactory)
         {
             _serverSession = new ServerSession();
 
-            _urlGroup = new UrlGroup(_serverSession, Logger);
+            _requestQueue = new RequestQueue(options.RequestQueueName, options.RequestQueueMode, Logger);
```

Comment:
I made this change because it made the code a bit cleaner IMO and makes more logical sense to me. I get that this may make it harder to back port changes though so if this is undesirable, it was the last commit so I can easily revert it.